tensorflow/lite/kernels/internal/BUILD

```diff
@@ -68,23 +68,6 @@ cc_library(
     ],
 )
 
-cc_library(
```

Comment:
Sorry, but do we need this change? I think we are not using portable_tensor_utils inside tflm since the latest change use micro_tensor_utils now?

Reply:
We do not use portable_tensor_utils any more so this change removes it.